### DISCUSSION TITLE
Feature request: Link app volume sliders across audio devices

### DISCUSSION THREAD

**Discussion by ntoff** [File-New-Project/EarTrumpet#1217] [Ideas]
It would be nice if I change the volume of an app on one audio device, for it to sync across all audio devices.

Perhaps an extra button in the right click menu for an application (this menu) would be nice in case for whatever reason volume syncing is undesirable for a particular application. Typically though, if I have an application at 30% for one audio interface, I have it at 30% for all.
![image](https://user-images.githubusercontent.com/9030085/187493863-08531d8b-127b-412c-9655-29b9daf25300.png)

**Comment by riverar** [comment]
Interesting idea, doesn't sound too bad. Does that app play on all devices?

**Reply by ntoff** [reply]
When you use the right click button to send an app to a different device, it would be nice if the volume level also carried over. That's probably the main use for this.

It might seem to make sense to control volume using in-app volume but certain games save volume on a per-save basis so I use ear trumpet to control the volume of those games so when I start a new game the volume is predictable.
